lib/src/dartdoc/index_to_pubdata.dart

```diff
@@ -37,12 +39,29 @@ PubDartdocData dataFromDartdocIndex(DartdocIndex index) {
     ));
   }
 
-  final documented = apiElements.where((e) => e.documentation != null).length;
+  /// Some symbols are repeated, and one instance may have a documentation, while
+  /// the other misses it. E.g. an enum with a default constructor may have dartdoc
+  /// on the enum type, but won't have anything on the (likely absent-from-code)
+  /// constructor.
```

Comment:
I'm not sure "symbols" are actually repeated, there **could** be dartdoc on the enum AND the constructor.

Given that the constructor will show up as a separate entity in the dartdoc, the question is if we should actually make this fix.

On the other hand it does not make much sense to document enum constructors separately...

Reply:
> On the other hand it does not make much sense to document enum constructors separately...

On top of that: these constructors are not in the code, they are omitted, but present in the dartdoc output. I think it would be a bad user experience to force developers to write constructors + documentation for those which could have been omitted otherwise.

Reply:
Updated documentation to reflect this better.